libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -244,6 +245,7 @@ def upsert_response(
         progress=progress,
         updated_at=updated_at or get_datetime(),
         job_runner_version=job_runner_version,
+        inc__attempts=1,
```

Comment:
Hmmmm... this would increment the number of attempts on every update, even if it's the first update for a new commit.
Something is still missing to get what we want:
- store the number of attempts for the current revision
- increment it on every new attempt on the current revision
- set it to 1 (if we keep the word attempt, or to 0 if we switch to retries, which I prefer as it's clearer to me: the default is 0, it's easy) when we set a new revision

Could we have tests in test_simple_cache.py to ensure it works as expected?

Reply:
I changed to "retries" and added a test to verify these scenarios:
- Firs time a cache in saved (retries=0)
- Second time a cache is save for the same revision (retries=1)
- Third time a cache is saved but for a different revision (retries=0)